### PR TITLE
Coming feedback BackEnd preliminary implementation

### PR DESCRIPTION
**Pull request by idigra**

### CODE REVIEW

File: backend/azure-functions/src/ComingFeedback/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        context.res = {
+            status: 400,
+            body: 'Bad request: missing reportId query parameter',
+        };
+
+        context.done();
+        return;
+    }
+
+    const authResult: FacebookAuth.AuthenticationResult = await FacebookAuth.authenticate(req, context);
```
Comment (YonatanKra) [inline backend/azure-functions/src/ComingFeedback/index.ts:21]: Can you set this up as a middleware?
It's supposed to be super simple with that: https://github.com/emanuelcasco/azure-middleware
This way you decouple the logic from the auth and can reuse the auth code everywhere (although we said we want to use the API mngmnt).

Comment (YonatanKra) [reply backend/azure-functions/src/ComingFeedback/index.ts:21]: Also, authentication should be first.

**Review by YonatanKra — APPROVED** [review]
Had one comment. Can set it up as a task for laters.

File: backend/azure-functions/src/ComingFeedback/index.ts (hunk 1 above)
Comment (idigra) [reply backend/azure-functions/src/ComingFeedback/index.ts:21]: Moved auth to be first.
Regards middleware - not part of this PR, it's infra auth change that might also conflict other PRs (that's why I splitted the auth changes for other PRs).

**Pull request merged**